homeassistant/components/nuki/config_flow.py

```diff
@@ -84,8 +103,14 @@ async def async_step_validate(self, user_input):
                     title=info["ids"]["hardwareId"], data=user_input
                 )
 
+        data_schema = (
+            USER_SCHEMA
+            if self.discovery_schema == {}
+            else vol.Schema(self.discovery_schema)
+        )
```

Comment:
```suggestion
        data_schema = self.discovery_schema or USER_SCHEMA
```

An empty `dict` is falsey

Reply:
Alternatively: If your style preference is to be explicit here and don't want to use an `or`, please convert the ternary to an if block since the code standard is to not merge ternary that span multiple lines

Reply:
The problem with the or is that it would be set as `self.discovery_schema` and not `vol.Schema(self.discovery_schema)`. Since USER_SCHEMA is already an schema an discovery_schema not. Otherwise I could change it up to set discovery_schema to an schema instead of an dict in the dhcp_flow

Reply:
You are are spot on. I missed that it was already wrapped.

Would you please convert the `ternary` to an `if` block to meet standards?

Reply:
Nevermind, I saw you changed the code already after I commented.